### PR TITLE
Category page title should be above the subtitle [SEED-680]

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@saeedmb8 @peterp 
Please review bug 11 - Category page: needs more styling. Title above subtitle and date should be added

**Comment by peterp** [comment]
@Mitso please add screenshots.

**Comment by peterp** [comment]
@Mitso When we looked at the current site all the dates were the same. Why do they want the dates included if the information isn't useful?

File: babycenter/static/css/modules/_article-list.scss [hunk 1]
```diff
@@ -13,7 +14,8 @@
 
 .article-list__title {
   font-weight: bold;
-  display: inline;
+  display: block;
```
Comment (peterp) [inline babycenter/static/css/modules/_article-list.scss]: This element should be block by default; so we can just remove this line.

File: babycenter/static/css/modules/_article-list.scss [hunk 2]
```diff
@@ -13,7 +14,8 @@
...
+
 
   &:after {
```
Comment (peterp) [inline babycenter/static/css/modules/_article-list.scss]: Do you think it still makes sense to add the colon `:` if the titles are not on the same line?

File: babycenter/static/css/modules/_article.scss [hunk 3]
```diff
@@ -11,7 +11,12 @@
 }
 
 .article__date {
-  color: #999;
+  color: #ccc;
+}
+
+.section-article_date {
```
Comment (peterp) [inline babycenter/static/css/modules/_article.scss]: perhaps we should rename `section-article_date` to something like `article__date--italic`

I don't know if italic is a good name though?

File: babycenter/static/css/modules/_article-list.scss (hunk 2 above)
Comment (Mitso) [inline babycenter/static/css/modules/_article-list.scss]: It doesn't make sense because it's a separator if it's one line. I will remove it.

**Comment by Mitso** [comment]
@peterp @saeedmb8 please review update.

Before 
develop-section

After 
local-section

**Comment by peterp** [comment]
👍

**Comment by Mitso** [comment]
@peterp @saeedmb8 a small change to be aware of:

Before 
old

After 
new

**Comment by smarzban** [comment]
👍

**Pull request merged**